needle/modules/dynamic/watch/traffic_capture.py

```diff
@@ -0,0 +1,42 @@
+from core.framework.module import BackgroundModule
+
+
+class Module(BackgroundModule):
+    meta = {
+        'name': 'Title',
+        'author': '@Andrea Amendola',
+        'description': 'Description',
+        'options': (
+            ('ip', '192.168.1.133', True, 'ip of the machine that will capture the traffic'),
+            ('port', '7575', True, 'port of the service that will handle the captured traffic'),
+            ('program', 'WIRESHARK', True, 'Select the program to use for viewing the captured traffic. Currently supported: WIRESHARK'),
```

Comment:
Correct me if I'm wrong but this doesn't appear to be used anywhere? If its not used I would remove this bit for now :).